models/Tool/UUID.php

```diff
@@ -138,7 +138,11 @@ public function createUuid()
             throw new \Exception('No instance identifier specified.');
         }
 
-        $this->uuid = (string) \Ramsey\Uuid\Uuid::uuid5(\Ramsey\Uuid\Uuid::NAMESPACE_DNS, $this->getInstanceIdentifier() . '~' . $this->getType() . '~' . $this->getItemId())->toString();
+        // namespace originally used from \Ramsey\Uuid\Uuid::NAMESPACE_DNS
+        $namespace = \Symfony\Component\Uid\Uuid::fromString('6ba7b810-9dad-11d1-80b4-00c04fd430c8');
+        $uuid = \Symfony\Component\Uid\Uuid::v5($namespace, $this->getInstanceIdentifier() . '~' . $this->getType() . '~' . $this->getItemId());
```

Comment:
```php
$uuid = \Symfony\Component\Uid\Uuid::v5(\Symfony\Component\Uid\Uuid::NAMESPACE_DNS, ...);
```

see https://github.com/ramsey/uuid/blob/master/src/Uuid.php#L50
Also, consider not using the fully qualified name, instead cleanup with `use` statements for better readability.

e.g.:
```php
use Symfony\Component\Uid\Uuid;
...
$uuid = Uuid::v5(Uuid::NAMESPACE_DNS, ...);
```

Reply:
I think PhpStorm did this automatically, since the class itself is also named `UUID`. 😉

Reply:
Oh, I see... nevermind then. ;)

Reply:
Just changed it manually in 23c898e27661eb7889fd97ebfc41d41d864a3e04 
Thanks!

Reply:
I just created #8224 for using the DNS namespace constant instead of the magic string.